### PR TITLE
Open experimental alignment services to public

### PR DESCRIPTION
**Pull request by philippotto**
image
image
image

### Open Questions
- should we mention that this feature might not be free to use in the future?
- should we limit the amount of sections that can be aligned for now?

### URL of deployed dev instance (used for testing):
- publicalignment.webknossos.xyz

### Steps to test:
- checking screenshots and wording is enough. if you want to see the modal on the dev instance, execute `webknossos.DEV.store.dispatch({ type: "SET_AI_JOB_MODAL_STATE", state: "neuron_inferral"  })` (no worker is configured for the dev instance)

### TODOs:
- [X] extend docs
- [X] adapt alignment banner
- [X] fine-tune styling
- [x] wait for RAM-optimized alignment worker
- [x] remove temporary code changes

### Issues:
- contributes #7955

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: .circleci/not-on-master.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
-if [ "${CIRCLE_BRANCH}" == "master" ]; then
+# if [ "${CIRCLE_BRANCH}" == "master" ]; then
```
Comment (hotzenklotz) [inline .circleci/not-on-master.sh:4]: Undo?

File: conf/application.conf [hunk 2]
```diff
@@ -146,7 +146,7 @@ features {
   discussionBoard = "https://forum.image.sc/tag/webknossos"
   discussionBoardRequiresAdmin = false
   hideNavbarLogin = false
-  isWkorgInstance = false
+  isWkorgInstance = true
```
Comment (hotzenklotz) [inline conf/application.conf:149]: Undo!

File: frontend/javascripts/admin/dataset/dataset_add_view.tsx [hunk 3]
```diff
@@ -238,15 +243,19 @@ const manualAnnotationBanner = (
     </div>
   </div>
 );
-const banners = [segmentationBanner, alignBanner, manualAnnotationBanner];
+const banners = [
+  // segmentationBanner,
```
Comment (hotzenklotz) [inline frontend/javascripts/admin/dataset/dataset_add_view.tsx:247]: Undo?

File: frontend/javascripts/admin/dataset/dataset_add_view.tsx [hunk 4]
```diff
@@ -238,15 +243,19 @@ const manualAnnotationBanner = (
...
+];
 
 function VoxelyticsBanner() {
   const [bannerIndex] = useState(Math.floor(Math.random() * banners.length));
   const theme = useSelector((state: OxalisState) => state.uiInformation.theme);
 
-  if (!features().isWkorgInstance) {
-    return null;
-  }
+  // if (!features().isWkorgInstance) {
```
Comment (hotzenklotz) [inline frontend/javascripts/admin/dataset/dataset_add_view.tsx:256]: Undo?

**Review by hotzenklotz — CHANGES_REQUESTED** [review]
There are various commented out sections that are likely meant to be undone before merging, right?

**Comment by philippotto** [comment]
> There are various commented out sections that are likely meant to be undone before merging, right?

yes, this was meant to make testing easier. if you are fine with the current state of the UI and wording, I'll undo the temporary changes.

**Review by hotzenklotz — APPROVED** [review]

**Pull request merged**